pyproject.toml

```diff
@@ -61,6 +61,7 @@ PyQt5 = "5.14.1"
 requests = "2.27.1"
 canmatrix = "0.9.1"
 click = "7.1.2"
+pylint-runner = "^0.6.0"
```

Comment:
This shouldn't be here.
```suggestion
```

Reply:
[4167a7f](https://github.com/epcpower/st/pull/48/commits/4167a7f05064dd583fa52044ee03ec872996049e)